# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wine/alsa.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The #define interface is a bit grave ...

  - Rene Rebe <rene@exactcode.de>

--- wine-20050524/dlls/winmm/winealsa/alsa.h.vanilla	2005-06-29 16:49:59.000000000 +0200
+++ wine-20050524/dlls/winmm/winealsa/alsa.h	2005-06-29 16:48:55.000000000 +0200
@@ -19,6 +19,7 @@
 # error You must include config.h to use this header
 #endif
 
+#undef interface
 #ifdef HAVE_ALSA_ASOUNDLIB_H
 #include <alsa/asoundlib.h>
 #elif defined(HAVE_SYS_ASOUNDLIB_H)
